.projenrc.ts

```diff
@@ -1,13 +1,15 @@
+import { ProjenStruct, Struct } from "@mrgrain/jsii-struct-builder";
 import { ReleasableCommits, awscdk } from "projen";
 import { ProseWrap } from "projen/lib/javascript";
 
 const project = new awscdk.AwsCdkConstructLibrary({
   author: "Ben Limmer",
   authorAddress: "hello@benlimmer.com",
-  cdkVersion: "2.1.0",
+  cdkVersion: "2.73.0", // Released in April 2023
```

Comment:
Since I'm using `jsii-struct-builder` to `extend RoleProps`, I'm bumping to a more modern minimum CDK version. This was released one year ago, so it feels reasonable to ask people to be on at least this version when upgrading.